examples/create_or_get_Stream.py

```diff
@@ -0,0 +1,23 @@
+from streamr.client.client import Client
+
+# To create or get a stream you should create a client at first
+myClient = Client({'apiKey': 'your-api-key'})
+
+# To create a new stream, you can use 'createStream' or 'getOrCreateStream' with a 'name' parameter
+# Note that createStream is running forcefully.
+# That means you can create two stream with same name but the streamIds are different
+
+
+#stream = client.createStream('stream-test')
+
+stream = myClient.getOrCreateStream('stream-test')
+
+# To get a stream, you can use 'getStreamByName' or getStreamById' method
+
+# getStreamByName will return all the streams with steam name
+# the return is a list object containning the information of all streams
+myClient.getStreamByName('stream-test')
+
+# get streamById will return the stream with the streamId
+# the return is a dictionary containning the information of the stream
+myClient.getStreamById('streamId')
```

Comment:
Test created new stream by failed to get "Fail to Get a Stream using its ID. The Status code: 404"

Reply:
We added a comment for the method. The error happens due to the wrong Stream ID. You should replace the streamId before using the method. The streamId can be found in the stream page, also can be obtained using the 'getStreamByName' method of which the response is shown as the following:
```
[{'config': {'fields': []},
  'dateCreated': '2019-01-27T09:16:22Z',
  'description': 'None',
  'feed': {'id': 7, 'module': 147, 'name': 'API'},
  'id': 'NQUihKiZTT6cc9F2-bLU2A',
  'lastUpdated': '2019-01-27T09:16:22Z',
  'name': 'stream-test',
  'partitions': 1,
  'requireSignedData': False,
  'uiChannel': False}]
```